test/aws/mzcompose.py

```diff
@@ -28,9 +28,11 @@
 NAMESPACE = ENVIRONMENT_NAME
```

Comment:
I renamed this composition to "aws" to reflect its new purpose as a general test of AWS-related functionality.